packages/shared/src/components/cards/SharedPostCardFooter.tsx

```diff
@@ -26,14 +26,14 @@ export const SharedPostCardFooter = ({
         isShort ? 'flex-row items-center' : 'flex-col',
       )}
     >
-      <div
+      <p
         className={classNames(
-          'flex text-theme-label-secondary typo-footnote',
```

Comment:
Does this not give issues for the short posts then?

Reply:
I did ran through multiple Squads, such as AI and Rush which have a wide variety of posts. They all seem to work okay.